src/main.rs

```diff
@@ -8,7 +8,52 @@ use yansi::{Color, Paint};
 
 use std::path::Path;
 
-use shelly::{Line, EmittedItem};
+use shelly::{Line, EmittedItem, lint::{Lint, self}};
+
+#[macro_use]
+extern crate structopt;
+
+use structopt::StructOpt;
+
+#[derive(StructOpt, Debug)]
+struct Opt {
+    /// Directory with code to analyze
+    #[structopt(long = "directory", default_value = ".")]
+    directory: String,
```

Comment:
It's not obvious, but it turns out that you can actually use `PathBuf` https://github.com/TeXitoi/structopt/issues/35#issuecomment-347271671